tasks.py

```diff
@@ -0,0 +1,86 @@
+import os
+import platform
+from datetime import datetime
+from pathlib import Path
+
+from robocorp import windows
+from robocorp.tasks import setup, task
+from robocorp.windows import WindowElement
+
+
+desktop = windows.Desktop()
+window = None  # Notepad window object after being controlled
+
+LOCATOR_NOTEPAD = 'class:"Notepad" subname:"Notepad"'
+OPEN_FILE = str(os.getenv("FILE_TO_OPEN", Path("devdata") / "test.txt").absolute())
+
+
+def get_win_version() -> str:
+    """Windows only utility which returns the current Windows major version."""
+    # Windows terminal `ver` command is bugged, until that's fixed, check by build
+    #  number. (the same applies for `platform.version()`)
+    version_parts = platform.version().split(".")
+    major = version_parts[0]
+    if major == "10" and int(version_parts[2]) >= 22000:
```

Comment:
I think it would be better if we could use some constants instead of magic numbers